src/web/app/components/user-notifications-list/user-notifications-list.component.spec.ts

```diff
@@ -84,7 +83,6 @@ describe('UserNotificationsListComponent', () => {
     fixture = TestBed.createComponent(UserNotificationsListComponent);
     notificationService = TestBed.inject(NotificationService);
     statusMessageService = TestBed.inject(StatusMessageService);
-    timezone = TestBed.inject(TimezoneService).guessTimezone();
```

Comment:
May I ask if it would be possible to put `component.timezone = timezone;` in `beforeEach` instead of adding the line in each method as it seems a bit repetitive? (Similarly for `admin-notifications-page.component.spec.ts`)

Reply:
Definitely possible